### PR TITLE
fix: remove retry button on incomplete proposal score

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Remove the button to trigger the score refresh, replaced by a background job.

Closes: #4485 

When the score is `pending`, instead of asking user to click on the button to trigger the score refresh, just show the message notifying the user that the scores is being finalized, while the request is being triggered in the background (on page load).

When the background score refresh is successful, replace the pending score message by the results.

When score is `invalid`, just keep showing the error message.

### How to test

1. Go to a closed proposal with pending score
2. The result box should display a message about score being finalizing
3. A few seconds later, the message should disappear, and be replaced by the final proposal results.

### To-Do

- [x] Do not refresh full page when scores are ready, refresh only the results

### CODE REVIEW

**Review by samuveth — APPROVED** [review]
Looks good, left one comment 

tACK

File: src/views/SpaceProposal.vue [hunk 1]
```diff
@@ -53,7 +57,8 @@ onMounted(() => {
       v-else-if="proposal && space"
       :space="space"
       :proposal="proposal"
-      @reload-proposal="loadProposal"
+      @reload-proposal="loadProposal()"
+      @soft-reload-proposal="loadProposal(true)"
```
[the remark this answers is not in the document]
Comment (wa0x6e) [reply src/views/SpaceProposal.vue:61]: How do we `emit('reloadProposal')` from childen, with `true` arg ?

[the remark this answers is not in the document]
Comment (wa0x6e) [reply src/views/SpaceProposal.vue:61]: Did not know we could pass argument to emit event, will look into that

**Comment by samuveth** [comment]
Just realized, could we also do softreload after the user voted. Currently whole page reloads, not sure it's needed

File: src/components/SpaceProposalPage.vue [hunk 2]
```diff
@@ -64,8 +64,8 @@ function clickVote() {
       : (modalOpen.value = true);
 }
 
-function reloadProposal() {
-  emit('reload-proposal');
+function reloadProposal(softReload = false) {
```
Comment (samuveth) [inline src/components/SpaceProposalPage.vue:67]:
```suggestion
function reloadProposal({ softReload = false } = {}) {
```
Maybe doing like this?

`reloadProposal(true)` isn't very readable IMO

**Comment by wa0x6e** [comment]
> Just realized, could we also do softreload after the user voted. Currently whole page reloads, not sure it's needed

In another pr, as it does not seems to work by just adding `true`

File: src/components/SpaceProposalPage.vue (hunk 2 above)
Comment (wa0x6e) [reply src/components/SpaceProposalPage.vue:67]: If we also soft reload votes later, we could just move the loading status toggler to  onmounted, and loadproposal will always be a soft reload

Comment (samuveth) [reply src/components/SpaceProposalPage.vue:67]: Ok we can do that. Should I merge this for now then?

Comment (wa0x6e) [reply src/components/SpaceProposalPage.vue:67]: Yes

**Pull request merged**